main.js

```diff
@@ -0,0 +1,65 @@
+var expenseCounter = function(num){
+  newNum = parseInt(num);
+  expenseTotal.textContent = newNum + count;
+  count = count + newNum;
+}
+
+var createRow = function(name, num){
+
+  if (name !== '' && num !== ''){
+      for(var count = 0;count < 1; count++){
+        var tr = document.createElement('tr');
```

Comment:
Typically when using `for` loops we assign the initialization variable to `i` for `initialize` or `index` etc.  After that we'd use `j`, `k`, etc...